api/src/main/java/javax/enterprise/inject/spi/Annotated.java

```diff
@@ -94,6 +113,8 @@
     /**
      * <p>
      * Determine if the program element has an annotation of a certain annotation type.
+     *
+     * If the <tt>annotationType</tt> is repeatable, then returns true if at least one instance is available
```

Comment:
I think you and Martin are contradicting each other.  I've been testing, `getAnnotation(SomeRepeatableAnnotation) == null` is true in these cases.  That's why I had added this to the javadoc, its not necessarily consistent with the JVM's impl.